hooks/pre_commit/20-certs_update.rb

```diff
@@ -0,0 +1,18 @@
+if module_enabled?('certs')
+  if app_value('certs_update_server_ca') && !module_enabled?('katello')
```

Comment:
I think app values are symbols
```suggestion
  if app_value(:certs_update_server_ca) && !module_enabled?('katello')
```

Reply:
Strings work, if you look at the code that was in the hooks that got moved.

Reply:
https://github.com/theforeman/kafo/blob/b7815c3af58c1fc42b6d561e16a2b6f086aad4c7/lib/kafo/hook_context.rb#L39-L44 suggests it does work, but is internally changed back to a symbol. However, this is the only file that actually uses strings. All others properly use symbols. IMHO we should always use symbols because it's more efficient and consistent.

Reply:
Would you mind filing a bug against Kafo to error if it's not a symbol? If we believe symbols are a better type, let's enforce it to prevent ambiguity.

Reply:
Opened https://projects.theforeman.org/issues/30781 and https://github.com/theforeman/kafo/pull/275

Reply:
Updated this PR to always use Symbol, to be consistent with our other usage in the Hooks.